controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/ApplicationSerializer.java

```diff
@@ -274,10 +280,13 @@ private ClusterInfo clusterInfoFromSlime(Inspector inspector) {
         String flavor = inspector.field(clusterInfoFlavorField).asString();
         int cost = (int)inspector.field(clusterInfoCostField).asLong();
         String type = inspector.field(clusterInfoTypeField).asString();
+        double flavorCpu = inspector.field(clusterInfoTypeField).asDouble();
```

Comment:
This should be `clusterInfoCpuField` ?
Same for mem and disk

Reply:
Great catch, I added unit test for this part and fixed the field names.